backend/de.metas.async/src/main/java/de/metas/async/processor/impl/planner/QueueProcessorPlanner.java

```diff
@@ -0,0 +1,360 @@
+/*
+ * #%L
+ * de.metas.async
+ * %%
+ * Copyright (C) 2022 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.async.processor.impl.planner;
+
+import com.google.common.collect.ImmutableList;
+import de.metas.async.api.WorkPackageLockHelper;
+import de.metas.async.model.I_C_Queue_WorkPackage;
+import de.metas.async.processor.IQueueProcessor;
+import de.metas.async.processor.QueuePackageProcessorId;
+import de.metas.async.processor.QueueProcessorId;
+import de.metas.async.processor.impl.AbstractQueueProcessor;
+import de.metas.lock.api.ILockManager;
+import de.metas.logging.LogManager;
+import de.metas.util.Services;
+import lombok.NonNull;
+import org.adempiere.ad.dao.IQueryBL;
+import org.adempiere.exceptions.AdempiereException;
+import org.adempiere.service.ISysConfigBL;
+import org.compiere.model.IQuery;
+import org.compiere.util.Env;
+import org.slf4j.Logger;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.Properties;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.locks.ReentrantLock;
+import java.util.stream.Collectors;
+
+public abstract class QueueProcessorPlanner implements Runnable
+{
+	private static final Logger logger = LogManager.getLogger(QueueProcessorPlanner.class);
+
+	private final static String SYSCONFIG_POLLINTERVAL = "de.metas.async.PollIntervallMillis";
```

Comment:
rename constant to SYSCONFIG_POLLINTERVAL_MILLIS.

as a general rule: pls always keep the "unit" near the name when using primitives like int or generic classes like BigDecimal.